### PR TITLE
[read] Add support for the 'gasp' table

### LINKED ISSUE #906: `gasp` table support needed in `read_fonts`
In order to read some hinting settings preferences from the font and configure hinting accordingly, we need access to the `gasp` table in `read_fonts`.

https://learn.microsoft.com/en-us/typography/opentype/spec/gasp

### PR DESCRIPTION
**Pull request by cmyr**
@drott is this fit for your purposes?

- closes #906

### CODE REVIEW

File: read-fonts/src/tables/gasp.rs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+//! The [gasp](https://learn.microsoft.com/en-us/typography/opentype/spec/gasp) table
+
+include!("../../generated/generated_cpal.rs");
```
Comment (rsheeter) [inline read-fonts/src/tables/gasp.rs:3]: s/cpal/gasp/

**Review by rsheeter — APPROVED** [review]
LGTM subject to addition of a test proving we can read at least one actual gasp as expected. Might need a new testdata item if we don't have any gasps?

**Comment by drott** [comment]
Thank you @cmyr, yes, this is exactly what I was looking for.

File: read-fonts/src/tables/gasp.rs (hunk 1 above)
Comment (cmyr) [reply read-fonts/src/tables/gasp.rs:3]: eek

**Pull request merged**